### PR TITLE
Scheduler: remove direct import to /pkg/api/core/v1/helper Part 1

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Scheduler: remove direct import to /pkg/api/core/v1/helper

**Which issue(s) this PR fixes**:

Ref #89930

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
none
```

### CODE REVIEW

File: pkg/scheduler/framework/plugins/helper/helpers.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import "k8s.io/api/core/v1"
+
+// GetPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
```
Comment (damemi) [inline pkg/scheduler/framework/plugins/helper/helpers.go:22]: Maybe add a comment that this came from `pkg/apis/core/v1/helper`. This function does seem to have a bunch of other uses through the codebase, but it's trivial enough that I don't see a problem duplicating it to remove a dependency.

Comment (gavinfish) [reply pkg/scheduler/framework/plugins/helper/helpers.go:22]: done

**Comment by YuikoTakada** [comment]
need to run ./hack/update-bazel.sh ?

**Comment by damemi** [comment]
Agree, this looks like it needs a `hack/update-bazel.sh` or `make update`

**Comment by gavinfish** [comment]
@damemi PTAL

**Review by damemi — COMMENTED** [review]
@gavinfish sorry this got put on hold, would you be interested in reviving and rebasing it?

**Comment by gavinfish** [comment]
> @gavinfish sorry this got put on hold, would you be interested in reviving and rebasing it?

@damemi sure, I happened to have some time for the community, let's work on it.

**Pull request closed without merging**